script/deployParameters/polygon-mumbai.json

```diff
@@ -3,6 +3,7 @@
   "weth9": "0x9c3C9283D3e44854697Cd22D3Faa240Cfb032889",
   "seaport": "0x00000000006c3852cbEf3e08E8dF289169EdE581",
   "seaportV2": "0x00000000000006c7676171937C444f6BDe3D6282",
+  "openseaConduit": "0x0000000000000000000000000000000000000000",
```

Comment:
exists here too - any chain with a non-0 seaport address needs a conduit address for this PR to work